src/status_im/ui/screens/privacy_and_security_settings/delete_profile.cljs

```diff
@@ -0,0 +1,86 @@
+(ns status-im.ui.screens.privacy-and-security-settings.delete-profile
+  (:require [status-im.ui.components.react :as react]
+            [quo.core :as quo]
+            [status-im.multiaccounts.core :as multiaccounts]
+            [status-im.ui.components.chat-icon.screen :as chat-icon.screen]
+            [re-frame.core :as re-frame]
+            [status-im.ui.components.topbar :as topbar]
+            [status-im.ui.components.styles :as components.styles]
+            [status-im.i18n :as i18n]
+            [reagent.core :as reagent]
+            [status-im.utils.security :as security]
+            [status-im.ui.screens.privacy-and-security-settings.events :as delete-profile]))
+
+(defn valid-password? [password]
+  (let [data (security/safe-unmask-data password)]
+    (and data (>= (count data) 6))))
+
+(defn keycard-pin []
+  #_(let [pin             @(re-frame/subscribe [:keycard/pin])
+          step              @(re-frame/subscribe [:keycard/pin-enter-step])
+          status            @(re-frame/subscribe [:keycard/pin-status])
+          pin-retry-counter @(re-frame/subscribe [:keycard/pin-retry-counter])
+          puk-retry-counter @(re-frame/subscribe [:keycard/puk-retry-counter])
+          error-label       @(re-frame/subscribe [:keycard/pin-error-label])]
+      [pin.views/pin-view
+       {:pin           pin
+        :status        status
+        :retry-counter pin-retry-counter
+        :error-label   error-label
+        :step          :current}]))
+
+(defn delete-profile []
+  (let [password       (reagent/atom nil)
+        text-input-ref (atom nil)]
+    (fn []
+      (let [keycard?     @(re-frame/subscribe [:keycard-multiaccount?])
+            multiaccount @(re-frame/subscribe [:multiaccount])
+            error        @(re-frame/subscribe [:delete-profile/error])]
+        (when (and @text-input-ref error (not @password))
+          (.clear ^js @text-input-ref))
+        [react/scroll-view components.styles/flex
+         [topbar/topbar {:modal? true}]
+         [react/view {:style {:flex            1
+                              :justify-content :space-between}}
+          [react/view {:style {:flex 1}}
+           [react/view {:style {:align-items :center}}
+            [quo/text {:weight :bold
+                       :size   :x-large}
+             (i18n/label :t/delete-profile)]]
+           [quo/list-item
+            {:title   (multiaccounts/displayed-name multiaccount)
+             :icon    [chat-icon.screen/contact-icon-contacts-tab
+                       (multiaccounts/displayed-photo multiaccount)]
+             :chevron true}]
+           [quo/text {:style {:margin-horizontal 24}
+                      :align :center
+                      :color :negative}
+            (i18n/label :t/delete-profile-warning)]
+           (if keycard?
+             [keycard-pin]
+             [quo/text-input
+              {:style             {:margin-horizontal 36
```

Comment:
Don't we have pre-defined styles for this sort of thing?

Reply:
would be nice, @Ferossgp any ideas?

Reply:
For now, we do not have a component which will manage spacing, but I'm looking at this approach https://github.com/mobily/stacks